src/app/pages/organizations/developers/developer/join.jsx

```diff
@@ -22,10 +24,47 @@ import { useFetchDevelopers, usePutJoinDevelopers } from 'api/developer';
 import { ChplActionBar } from 'components/action-bar';
 import { ChplTextField } from 'components/util';
 import { getAngularService } from 'services/angular-react-helper';
+import ClearIcon from '@material-ui/icons/Clear';
+import AddIcon from '@material-ui/icons/Add';
+import { theme } from 'themes';
 
 const useStyles = makeStyles({
   headingPadding: {
     padding: '16px 0',
+    backgroundColor: '#fff',
+  },
+  boxContainer: {
+    padding: "32px 32px",
```

Comment:
All of the `"` should be `'`, as per our JS convention. I think the ESLINT tool should have warned you about these